assets/src/edit-story/elements/image/karma/edit.karma.js

```diff
@@ -130,6 +130,20 @@ describe('Image Editor', () => {
 
         await fixture.snapshot('Image reset');
       });
+
+      it('should allow image to be scaled using mouse-wheel', async () => {
+        // Validate that image has focus
+        const image = fixture.editor.canvas.editLayer.media;
+        expect(image).toHaveFocus();
+
+        // Scale image up by scrolling three steps
+        await fixture.events.mouse.moveRel(image, 10, 10);
+        await fixture.events.mouse.wheel({ deltaY: 1 });
+        await fixture.events.mouse.wheel({ deltaY: 1 });
+        await fixture.events.mouse.wheel({ deltaY: 1 });
+
+        await fixture.snapshot('Image scaled up');
```

Comment:
Could we perhaps test that the selected element has zoom other than the default now?
The snapshot verifications are easy to slip through, since we can, maybe let's test it in a more robust way.

Reply:
Added in 31b4d306f.